Doc/c-api/refcounting.rst

```diff
@@ -97,6 +97,39 @@ objects.
    :c:func:`Py_DECREF`, and the same warning applies.
 
 
+.. c:function:: PyObject* _Py_Borrow(PyObject *o)
+
+   :term:`Borrow a reference <borrowed reference>` to an object: decrement the
+   object reference count and return the object.
+
+   The object must not be ``NULL``; if you aren't sure that it isn't
+   ``NULL``, use :c:func:`_Py_XBorrow`.
+
+   This function can be used as an expression to update an old C extension to
```

Comment:
I would write "This function is intended to be used when updating an old C extension ..." to make it clearer that this is the only intended use case for `_Py_Borrow` and not one of its many possible use cases.

Reply:
Done